lib/gollum-lib/version.rb

```diff
@@ -1,5 +1,5 @@
 module Gollum
   module Lib
-    VERSION = '4.2.4'
+    VERSION = '4.2.5'
```

Comment:
Another problem with rebasing to the `5.x` branch. Version [should be](https://github.com/gollum/gollum-lib/blob/gollum-lib-5.x/lib/gollum-lib/version.rb#L3) `'5.0.a.3'`.